services/vehicle-assembly-process-defect-detection-model-service/app/routers/model.py

```
@@ -0,0 +1,196 @@
+                "success": False,
+                "message": "모델 성능 조회 중 오류가 발생했습니다",
+                "request_id": request_id,
+                "timestamp": datetime.now().isoformat()
+            }
+        )
+
+
+@router.post("/reload")
+async def reload_model(
+    request_id: str = Depends(get_request_id),
+    model_service: ModelService = Depends(get_model_service)
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)